app/views/layouts/admin_layout.html.erb

```diff
@@ -38,11 +38,11 @@
       app_name: "Collections Publisher",
       phase: "beta"
     } %>
-    <% if yield(:back_link).present? %>
-      <%= render "govuk_publishing_components/components/back_link", href: yield(:back_link) %>
```

Comment:
So you don't need to render the `back_link` component? Is that included somewhere else?

Reply:
We're already rendering it [in each template file](https://github.com/alphagov/collections-publisher/blob/master/app/views/steps/edit.html.erb#L18). What we are doing before this PR is that we were testing if anything gets passed from the template file to the layout using `content_for` but instead of using that block we were re-creating it in the layout and passed that block as `href`. The reason why this didn't failed is because we're [not using the back_link block, but the breadcrumbs one](https://github.com/alphagov/collections-publisher/blob/master/app/views/steps/edit.html.erb#L17). Let me know if this doesn't make sense, we can discuss it offline.

Reply:
Thanks for explaining.